modules/local/mhcflurry_predictneoepitopesclass1.nf

```diff
@@ -1,24 +1,11 @@
-// Import generic module functions
-include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
-
-params.options = [:]
-options        = initOptions(params.options)
-
 process MHCFLURRY_PREDICTNEOEPITOPESCLASS1 {
     tag "$meta"
     label 'process_low'
 
-    publishDir "${params.outdir}",
-        mode: params.publish_dir_mode,
-        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:'class_1_bindings', publish_id:'class_1_bindings') }
-
-    conda (params.enable_conda ? "bioconda::mhcflurry=1.4.3" : null)
-    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/mhcflurry:1.4.3--py_0"
-    } else {
-        container "quay.io/biocontainers/mhcflurry:1.4.3--py_0"
-    }
-
+    conda (params.enable_conda ? "bioconda::fred2=2.0.6 bioconda::mhcflurry=1.4.3 bioconda::mhcnuggets=2.3.2" : null)
```

Comment:
there would be a container with `fred2=2.0.7` which we use in epitopeprediction. Just in case you missed that.

Reply:
You are right, maybe I should update the mulled container 👍